packages/appium-tizen-tv-driver/lib/rc-pair.js

```diff
@@ -16,6 +16,12 @@ export async function pairRemote({host, port}) {
     throw err;
   }
 
+  // Possible port detection
+  const deviceData = await got.get(`http://${host}:8001/api/v2/`).json();
+  if (deviceData?.device?.TokenAuthSupport !== 'true' && port === 8002) {
```

Comment:
Or, in case the response does not include `TokenAuthSupport` completely, it may not require "token". The websocket should keep 8001